cluster/src/main/java/org/apache/iotdb/cluster/client/sync/SyncDataClient.java

```diff
@@ -56,9 +55,9 @@ public SyncDataClient(TProtocolFactory protocolFactory, Node node, SyncClientPoo
         protocolFactory.getProtocol(
             RpcTransportFactory.INSTANCE.getTransport(
                 new TSocket(
-                    node.getInternalIp(),
-                    node.getDataPort(),
-                    RaftServer.getConnectionTimeoutInMS()))));
+                    node.getInternalIp(), node.getDataPort() // ,
+                    //                    RaftServer.getConnectionTimeoutInMS()
```

Comment:
if this parameter does not use, could remove... Try not to comment

Reply:
Why remove the connection timeout parameter?

Reply:
> if this parameter does not use, could remove... Try not to comment

yes, It‘s just a temp step in order to run quickly, need to be improved next according to the detail spec.